store/migration/10001__init_schema.sql

```diff
@@ -416,6 +416,7 @@ CREATE TABLE db (
     project_id INTEGER NOT NULL REFERENCES project (id),
     -- If db is restored from a backup, then we will record that backup id. We can thus trace up to the original db.
     source_backup_id INTEGER REFERENCES backup (id) ON DELETE SET NULL,
+    schema_version TEXT NOT NULL,
```

Comment:
nit: Put this after last_successful_sync_ts and comment it's synced from the migration history table on the db instance?